03_module/Makefile

```diff
@@ -0,0 +1,37 @@
+BBB=1
+LOCAL=0
+TARGET ?= LOCAL
+
+ifneq ($(KERNELRELEASE),)
+
+# kbuild part of makefile
+obj-m := first_module.o
+
+else
+
+ifeq ($(TARGET),BBB) 
+
+REMDIR = /srv/nfs/busybox/home
+KDIR = ~/Projects/repos/linux/kernel/linux-stable/
```

Comment:
Removing of `?` may help when troubleshooting bot now
```bash
export TARGET=BBB
export KDIR=~/GL-LKT/repos/linux-stable/
make
```
does not work. Only this way is possible:
```bash
export TARGET=BBB
make KDIR=~/GL-LKT/repos/linux-stable/
```
It's not a big deal for this homework but read more about env / command line assignments priorities in makefiles.